Client/Frontend/Browser/Search/BraveSearchPromotionCell.swift

```diff
@@ -9,8 +9,9 @@ import BraveUI
 import BraveShared
 
 class BraveSearchPromotionCell: UITableViewCell {
-  
-  struct UX {
+  private struct DesignUX {
+    static let paddingX: CGFloat = 15.0
+    static let paddingY: CGFloat = 10.0
```

Comment:
`CGFloat` type declaration can be skipped now if I remember correctly

Reply:
yeap at https://github.com/brave/brave-ios/pull/5678/commits/88ddc0b1c93d257d766916c23cb4d1dd0423a6fd